### ISSUE TITLE
JIT causing build to fail when I enable @tailwindcss/forms

### ISSUE THREAD

**Issue by lukebennett88** [tailwindlabs/tailwindcss-jit#158]
### What version of @tailwindcss/jit are you using?

0.1.10

### What version of Node.js are you using?

v14.4.0

### What browser are you using?

N/A

### What operating system are you using?

macOS 11.2.3

### Reproduction repository

https://github.com/lukebennett88/tailwind-jit-bug

For some reason, whenever I enable the [`@tailwindcss/forms`](https://github.com/tailwindlabs/tailwindcss-forms) plugin, my Gatsby server crashes.

This happens when running both `gatsby develop` and `gatsby build`.

I have another Gatsby site that is very similar using the same plugins with no issues, so not sure why this is happening (maybe incompatibility with different versions of the packages I'm using?)

Here is the error I see in my terminal:

```
 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed

Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
TypeError: Cannot read property 'type' of null
    at extractCandidates (.../node_modules/@tailwi
ndcss/jit/src/lib/setupContext.js:314:22)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:328:22
    at Array.flatMap (<anonymous>)
    at withIdentifiers (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:326:30)
    at addBase (.../node_modules/@tailwindcss/jit/
src/lib/setupContext.js:390:38)
    at .../node_modules/@tailwindcss/forms/src/ind
ex.js:252:5
    at registerPlugins (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:580:7)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:770:5
    at .../node_modules/@tailwindcss/jit/src/index
.js:37:49
    at LazyResult.runOnRoot (.../node_modules/post
css/lib/lazy-result.js:303:16)
    at LazyResult.runAsync (.../node_modules/postc
ss/lib/lazy-result.js:355:26)
    at LazyResult.async (.../node_modules/postcss/
lib/lazy-result.js:205:30)
    at LazyResult.then (.../node_modules/postcss/l
ib/lazy-result.js:190:17)

File: .cache/blank.css


 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed

Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
TypeError: Cannot read property 'type' of null
    at extractCandidates (.../node_modules/@tailwi
ndcss/jit/src/lib/setupContext.js:314:22)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:328:22
    at Array.flatMap (<anonymous>)
    at withIdentifiers (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:326:30)
    at addBase (.../node_modules/@tailwindcss/jit/
src/lib/setupContext.js:390:38)
    at .../node_modules/@tailwindcss/forms/src/ind
ex.js:252:5
    at registerPlugins (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:580:7)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:770:5
    at .../node_modules/@tailwindcss/jit/src/index
.js:37:49
    at LazyResult.runOnRoot (.../node_modules/post
css/lib/lazy-result.js:303:16)
    at LazyResult.runAsync (.../node_modules/postc
ss/lib/lazy-result.js:355:26)
    at LazyResult.async (.../node_modules/postcss/
lib/lazy-result.js:205:30)
    at LazyResult.then (.../node_modules/postcss/l
ib/lazy-result.js:190:17)

File: src/styles/global.css

failed Building development bundle - 7.974s
ERROR in ./.cache/blank.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
TypeError: Cannot read property 'type' of null
    at extractCandidates (.../node_modules/@tailwi
ndcss/jit/src/lib/setupContext.js:314:22)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:328:22
    at Array.flatMap (<anonymous>)
    at withIdentifiers (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:326:30)
    at addBase (.../node_modules/@tailwindcss/jit/
src/lib/setupContext.js:390:38)
    at .../node_modules/@tailwindcss/forms/src/ind
ex.js:252:5
    at registerPlugins (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:580:7)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:770:5
    at .../node_modules/@tailwindcss/jit/src/index
.js:37:49
    at LazyResult.runOnRoot (.../node_modules/post
css/lib/lazy-result.js:303:16)
    at LazyResult.runAsync (.../node_modules/postc
ss/lib/lazy-result.js:355:26)
    at LazyResult.async (.../node_modules/postcss/
lib/lazy-result.js:205:30)
    at LazyResult.then (.../node_modules/postcss/l
ib/lazy-result.js:190:17)
    at processResult (.../node_modules/webpack/lib
/NormalModule.js:675:19)
    at
.../node_modules/webpack/lib/NormalModule.js:774:5
    at .../node_modules/loader-runner/lib/LoaderRu
nner.js:399:11
    at .../node_modules/loader-runner/lib/LoaderRu
nner.js:251:18
    at context.callback (.../node_modules/loader-r
unner/lib/LoaderRunner.js:124:13)
    at Object.loader (.../node_modules/postcss-loa
der/dist/index.js:104:7)
 @ ./.cache/app.js 26:0-21

ERROR in ./src/styles/global.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
TypeError: Cannot read property 'type' of null
    at extractCandidates (.../node_modules/@tailwi
ndcss/jit/src/lib/setupContext.js:314:22)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:328:22
    at Array.flatMap (<anonymous>)
    at withIdentifiers (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:326:30)
    at addBase (.../node_modules/@tailwindcss/jit/
src/lib/setupContext.js:390:38)
    at .../node_modules/@tailwindcss/forms/src/ind
ex.js:252:5
    at registerPlugins (.../node_modules/@tailwind
css/jit/src/lib/setupContext.js:580:7)
    at .../node_modules/@tailwindcss/jit/src/lib/s
etupContext.js:770:5
    at .../node_modules/@tailwindcss/jit/src/index
.js:37:49
    at LazyResult.runOnRoot (.../node_modules/post
css/lib/lazy-result.js:303:16)
    at LazyResult.runAsync (.../node_modules/postc
ss/lib/lazy-result.js:355:26)
    at LazyResult.async (.../node_modules/postcss/
lib/lazy-result.js:205:30)
    at LazyResult.then (.../node_modules/postcss/l
ib/lazy-result.js:190:17)
    at processResult (.../node_modules/webpack/lib
/NormalModule.js:675:19)
    at
.../node_modules/webpack/lib/NormalModule.js:774:5
    at .../node_modules/loader-runner/lib/LoaderRu
nner.js:399:11
    at .../node_modules/loader-runner/lib/LoaderRu
nner.js:251:18
    at context.callback (.../node_modules/loader-r
unner/lib/LoaderRunner.js:124:13)
    at Object.loader (.../node_modules/postcss-loa
der/dist/index.js:104:7)
 @ ./gatsby-browser.js 9:0-33
 @ ./.cache/api-runner-browser-plugins.js 22:10-41
 @ ./.cache/api-runner-browser.js 4:14-53
 @ ./.cache/app.js 13:0-65 28:0-70 32:20-29 45:0-14 99:17-26 144:8-17 28:0-70

2 ERRORS in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
webpack compiled with 4 errors
```

Thanks for taking a look!

**Comment by veerman72** [comment]
I have the same problem when using **@tailwindcss/jit** v0.1.10 with **@tailwindcss/forms** v0.3.0. 
Downgrading to @tailwindcss/forms v0.2.1 is the workaround.

**Comment by vinayakkulkarni** [comment]
Yep, can confirm the same, Issue is with forms v0.3.0, downgrading to v0.2.1 works

I've opened an issue in Nuxt tailwindcss-module as well - https://github.com/nuxt-community/tailwindcss-module/issues/320

**Comment by bradlc** [comment]
Thanks for reporting! Should be fixed in version `0.3.2` of the forms plugin 👍
